src/video-player.utils.js

```diff
@@ -9,6 +9,7 @@ import {
   AUTO_PLAY_MODE
 } from './video-player.const';
 import { isString } from './utils/type-inference';
+require('css.escape');
```

Comment:
Please change it to:
`import cssEscape from 'css.escape';`

Reply:
Also please add a comment about it, something like:  'includes a polyfill to support ie11'

Reply:
Done